specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceCreateIndex.json

```diff
@@ -112,6 +112,15 @@
                     "tempuri.org"
                 ],
                 "maxAgeInSeconds": 60
+            },
```

Comment:
I'm wondering if we should have a parallel set of examples for `Create`/`CreateOrUpdate`/`Get` with encryption, since it will not be a mainstream scenario. I'm not sure what the consequences are of having multiple examples for an operation in the Swagger spec (hard to tell since we don't generate our REST API docs from Swagger yet, but we will in the future). I'll look into this and try to get some guidance, but in the meantime this should be fine.

@HeidiSteen Do you know what happens in terms of generated docs if a Swagger spec has more than one example for an operation?

Reply:
That's what I originally did, as seen in this commit -> https://github.com/Azure/azure-rest-api-specs/pull/5567/commits/75f1d081e7cce46c354a6dc0f0bc55cd2646473d

However, once I was done, I realized that every single example needs to be associated with a specific rest API declaration (one example per API), and since we don't have any specific "encrypted create/update/get" API endpoint, I couldn't find any place where to actually link to those example. And the "Avocado" validation step fails if you have unreferenced samples.

Reply:
It should be possible to have multiple examples per operation. At least, the schema for x-ms-examples seems to suggest that this is possible. Whether it's a good idea is a whole other question.